src/common/exif.cc

```diff
@@ -996,7 +996,7 @@ int dt_exif_get_thumbnail(const char *path, uint8_t **buffer, size_t *size, char
     Exiv2::PreviewPropertiesList list = loader.getPreviewProperties();
     if(list.empty())
     {
-      std::cerr << "[exiv2] couldn't find thumbnail for " << path << std::endl;
+      // std::cerr << "[exiv2] couldn't find thumbnail for " << path << std::endl;
```

Comment:
hmm, i do agree that this line is not too useful
But, i also do not think it should be just removed
change it to:
```
dt_print(DT_DEBUG_LIGHTTABLE, "[exiv2] couldn't find thumbnail for %s", path);
```
These embedded thumbs are mostly specific to lighttable, so i guess this is the best place.